src/integrationTest/java/com/kryptokrauts/aeternity/generated/api/ZIndaexServiceTest.java

```diff
@@ -0,0 +1,20 @@
+package com.kryptokrauts.aeternity.generated.api;
+
+import com.kryptokrauts.aeternity.sdk.service.indaex.domain.StatusResult;
+import io.vertx.ext.unit.TestContext;
+import org.junit.Test;
+
+public class ZIndaexServiceTest extends BaseTest {
+
+  @Test
+  public void testStatus(TestContext context) {
+    this.executeTest(
+        context,
+        t -> {
+          StatusResult statusResult = this.aeternityServiceNative.indaex.blockingGetStatus();
+          _logger.info(statusResult.toString());
+          context.assertEquals("5.8.0", statusResult.getNodeVersion());
+          context.assertEquals("1.0.3", statusResult.getMdwVersion());
```

Comment:
won't the test fail on future versions?

Reply:
well we always need to adjust the test when we update the image. so it depends :D